src/outputs/tau.py

```diff
@@ -21,6 +21,8 @@ def output_tau(
     cora_mapper: pd.DataFrame,
     postcode_itl_mapper: pd.DataFrame,
     pg_alpha_num: pd.DataFrame,
+    pg_num_alpha: pd.DataFrame,
+    sic_pg_alpha: pd.DataFrame,
```

Comment:
Do we use these mappers in the function?

Reply:
no, we don't. Removed